### DISCUSSION TITLE
Tesseract-OCR

### DISCUSSION THREAD

**Discussion by Kelthuzer** [Kyrluckechuck/TFT-Bot#153] [Q&A] [answered]
I can't figure out how to activate it in the bot. there is no mention of this in the settings file

**Comment by Kyrluckechuck** [comment, marked as answer]
Hi there,

Which version of the bot are you running? Is it a tagged release, or the source code cloned from the repo?

Have you tried the steps outlined in the [README](https://github.com/Kyrluckechuck/TFT-Bot#optional-install-tesseract-ocr--enable-economic-decision-making)?

Specifically:
> After you have downloaded and installed Tesseract-OCR, set `mode` under `economy` to `ocr_standard` in the `config.yaml`

**_Edit: Resolved by running as adminstrator_**

**Reply by Kyrluckechuck** [reply]
On that note, I realize that unless you had been running the source, you were likely running a release that did not have this change included. Please try again with the [latest release, v0.11.9](https://github.com/Kyrluckechuck/TFT-Bot/releases/tag/v0.11.9.14)

**Reply by Kelthuzer** [reply]
i use .exe, after update the config file is not created by itself and is ignored in %appdata%\tft bot\, strings appeared in TFT Bot\_MEI****\tft_bot\resources but _mei new each start

**Reply by Kyrluckechuck** [reply]
I've created an issue at https://github.com/Kyrluckechuck/TFT-Bot/issues/154, if you're able to attach your logs & configuration to that issue and we'll try to help debug what's going wrong there! 😄

**Reply by Kelthuzer** [reply]
I would be glad but the /tft bot folder is completely empty

**Reply by Kyrluckechuck** [reply]
Odd. Can you check if the folder you are executing the bot in may have an `output` folder that was created?

**Reply by Kelthuzer** [reply]
folder output creat in %appdata%\TFT Bot\_MEIXXXXX\output
no output folder near .exe

**Reply by akshualy** [reply]
Chipping in here, not sure why your logs are not being created. What you however **can** do, is 

1. Go to the folder where your TFT bot .exe is
2. Hold Shift and then right-click in an empty area of that folder
3. Click `Open PowerShell window here` in the context menu that opened
4. Run `.\TFT.Bot.exe -v` - This will run the bot in verbose mode, meaning more information will be logged
5. Let the bot run for a while (at least until you're fully loaded into a game)
6. Go into the bot console window
7. Press CTRL + A
8. Press CTRL + C
9. Paste the contents at a [pasting service](https://pastebin.com) and send the link here, or paste the contents into a file and upload the file here.

That should result in all the logs we could get regardless of whether it was created in the TFT Bot folder.

**Reply by Kelthuzer** [reply]
https://pastebin.com/FRdecU4E

**Reply by akshualy** [reply]
Looks like there are no issues logged here 🤔 Yea sorry, I have no idea why it's not able to create the config or the logs in your app data folder, especially if it's able to create the _MEI folder. And without the config, it is currently not possible to set the economy mode to OCR.

We should probably add a debug line about what the storage path is, and also check and log if the config file was created @Kyrluckechuck

**Reply by Kyrluckechuck** [reply]
@Gadsee I would 100% agree, created an issue for that to be added [here](https://github.com/Kyrluckechuck/TFT-Bot/issues/157).

Regarding this specific issue, I tested it with the [v0.11.9 release executable](https://github.com/Kyrluckechuck/TFT-Bot/releases/download/v0.11.9.14/TFT.Bot.exe), and it is creating the folder at `C:\Users\Justin\AppData\Roaming\TFT Bot` without issue, including the `config.yaml` and `tft-bot-debug[..].txt` log 🤔

**Reply by Kelthuzer** [reply]
i found sulution =) "run as admin"
